llumnix/backends/vllm/llm_engine.py

```diff
@@ -35,16 +35,19 @@
 from llumnix.backends.profiling import LatencyMemData
 from llumnix.server_info import ServerInfo
 from llumnix.internal_config import MigrationConfig
-from llumnix.rpc.queue_client import QueueClient
+from llumnix.output_queue.output_queue_client_base import OutputQueueClientBase
+from llumnix.output_queue.utils import get_output_queue_client
 
 logger = init_logger(__name__)
 
 
 class AsyncPutQueueThread(threading.Thread):
-    def __init__(self, instance_id):
+    def __init__(self, instance_id, output_queue_type):
```

Comment:
We can give data type hint, instance: str, output_queue_type: xxx...

Reply:
get